src/main/java/com/blackducksoftware/integration/hub/alert/channel/email/EmailChannel.java

```diff
@@ -97,14 +97,11 @@ public void handleEvent(final EmailEvent emailEvent) {
     }
 
     @Override
-    public String testMessage(final GlobalEmailConfigEntity globalConfig) {
-        if (globalConfig != null) {
-            final ProjectData data = new ProjectData(DigestTypeEnum.REAL_TIME, "Test Project", "Test Version", Collections.emptyMap());
-            final String emailAddress = globalConfig.getMailSmtpFrom();
-            sendMessage(Arrays.asList(emailAddress), new EmailEvent(data, null), globalConfig);
-            return "Attempted to send message with the given configuration.";
-        }
-        return null;
+    public String testMessage() {
+        final ProjectData data = new ProjectData(DigestTypeEnum.REAL_TIME, "Test Project", "Test Version", Collections.emptyMap());
+        final String emailAddress = getGlobalConfigEntity().getMailSmtpFrom();
```

Comment:
Again, the more I look at this, the more I think the testMessage will only relate to the distribution config. I don't know if it makes sense to send a test email to the same address it was sent from.

Reply:
The only email address we have at this point is the one configured as the From address, so we must send the test email to that address OR we need to figure out if we want to add an additional field for the Test only OR should we use the email address of the Hub user configured.

Reply:
I like the last option because that will be consistent with testing the other channels.